### ISSUE TITLE
Move embedded python to separate file, make sure it's linted that linting is added to CI.

### ISSUE THREAD

**Issue by tublitzed** [mozilla-services/syncstorage-rs#955]
Right now we have some [embedded python](https://github.com/mozilla-services/syncstorage-rs/blob/master/src/web/tokenserver.rs#L141) as a string in our Rust logic. This would be really easy to accidentally introduce say, a syntax error into, and we wouldn't catch it as-is.

Let's isolate the python into it's own file so it can be reviewed/linted/tested/etc more easily.

**Comment by ethowitz** [comment]
No longer needed after #1049
